### PR TITLE
[13.0][MIG] vault, vault_share

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: vault/views/assets.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                href="/vault/static/src/scss/vault.scss"
+            />
+        </xpath>
+    </template>
+
+    <template id="tests_assets" inherit_id="web.tests_assets">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/vault/static/tests/vault_tests.js" />
+        </xpath>
+    </template>
```
Comment (fkantelberg) [inline vault/views/assets.xml:31]: Isn't supported in 13.0 and was commented out before.

File: vault/views/vault_entry_views.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                />
+                <filter
+                    string="Vault"
+                    name="vault"
+                    domain="[]"
+                    context="{'group_by': 'vault_id'}"
+                />
+                <searchpanel>
+                    <field name="parent_id" string="Entries" enable_counters="1" />
+                </searchpanel>
```
Comment (fkantelberg) [inline vault/views/vault_entry_views.xml:168]: searchpanel is also only 14.0+

**Review by fkantelberg — CHANGES_REQUESTED** [review]
I tested the main functions and all seems to work good when you apply the 2 requested changes. Otherwise there is only the possibility that a user can get locked out of the vault when the login is changed.

File: vault/views/vault_entry_views.xml (hunk 2 above)
Comment (rousseldenis) [reply vault/views/vault_entry_views.xml:168]: Nope, I found them in 13, but attribute enable_counters not.

**Review by CarlosRoca13 — COMMENTED** [review]
I have tested the migration locally and everything seems to be working perfectly except that users without sharing permissions can perform this action. Can you check it?

**Comment by fkantelberg** [comment]
@CarlosRoca13 
What do you mean with action `share`? The share permission is allowing users to add additional users to the vault (to share the vault with a different user). It isn't blocking a user from sharing secrets using the inbox or the external shares.

**Review by CarlosRoca13 — APPROVED** [review]
Ohh!! Ok, so it's working as expected.

Thanks for the clarification

**Review by fkantelberg — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
So please rebase and run pre-commit :sweat_smile:

**Comment by pedrobaeza** [comment]
@rousseldenis can you fix pre-commit? Or do we supersede it respecting your authorship?

**Comment by rousseldenis** [comment]
> @rousseldenis can you fix pre-commit? Or do we supersede it respecting your authorship?

Yes I will do it.

Was on dad-holidays :sweat_smile:

**Comment by pedrobaeza** [comment]
Hey, congratulations! We have superseded meanwhile the PR in #386. Do you mind? Or do you prefer to continue in this one?

**Comment by rousseldenis** [comment]
Nope, that's great! Thanks

**Pull request closed without merging**